lab2/kursinis.tex

```diff
@@ -47,10 +47,29 @@ \section{Context}
 the educational experience for both students and teachers. With a user-friendly interface and a comprehensive set of features, EduPal aims to streamline the learning process, fostering a more efficient and engaging educational environment.
 
 \subsection{Functionality overview}
-List features here
 
-\subsection{Tech stack}
-List tech stack here
+At present, EduPal has these functionalities:
+
+\begin{itemize}
+    \item \textbf{Create and Manage Subjects}: Users have the ability to create new subjects and manage existing ones within the system. This includes creating topics under each subject and uploading notes in PDF format related to those topics.
+    \item \textbf{Note Sharing and Interaction}: Users can interact with notes shared by others by viewing, rating, and engaging in discussions within the topic's section. The system allows users to rate notes based on usefulness to promote the most valuable ones. Users also have options to export notes to a PDF section, download them to their device, or save them as drafts for further editing. 
+    \item \textbf{Pomodoro Timer System}: The system features a Pomodoro timer system designed to enhance productivity. Users can customize their study and break times according to their preferences. The timer notifies users when it's time to take a break, helping them maintain focus and manage their study sessions effectively.
+    \item \textbf{Goal Setting}: Users can set study goals within the system, specifying desired subjects and study times. This feature enables users to plan and organize their study sessions more efficiently, helping them stay on track with their academic or learning objectives.
```

Comment:
More detailed explanation is helpful. I don't get from this what goals do and how users take advantage of these.

Reply:
"specifying desired subjects and study times" And how is it helpful in the end? What are outcomes?